Sources/Authentication/AppAuthSession.swift

```diff
@@ -40,7 +40,8 @@ public final class AppAuthSession: LoginSession {
         
         let config = OIDServiceConfiguration(
             authorizationEndpoint: configuration.authorizationEndpoint,
-            tokenEndpoint: URL(string: "https://oidc.integration.account.gov.uk/token")!)
+            tokenEndpoint: configuration.tokenEndPoint
```

Comment:
please can we be consistent with naming ? `Endpoint` or `endpoint`

Reply:
eek yes sorry, my bad 🙂